packages/components/src/components/text-field-wrapper/text-field-wrapper.tsx

```diff
@@ -145,7 +145,9 @@ export class TextFieldWrapper {
 
   @Watch('showCounter')
   public onShowCounterChange(): void {
-    this.updateCounterVisibility();
+    if (this.input) {
+      this.updateCounterVisibility();
```

Comment:
As alternative we could also adjust the function behind which causes the issue, e.g. by extending it from `export const hasCounter = (el: HTMLTextAreaElement | HTMLInputElement): boolean => el.maxLength >= 0;` to `export const hasCounter = (el: HTMLTextAreaElement | HTMLInputElement): boolean => el && el.maxLength >= 0;`

Reply:
Thanks for the suggestion. We would need to change ist as follows:

```tsx
// Before
export const hasCounterAndIsTypeText = (el: HTMLInputElement): boolean => isType(el.type, 'text') && hasCounter(el);
export const hasUnitAndIsTypeTextOrNumber = ({ type }: HTMLInputElement, unit: string): boolean => {
  return !!unit && (isType(type, 'text') || isType(type, 'number'));
};
```

```tsx
// After
export const hasCounterAndIsTypeText = (el: HTMLInputElement): boolean => el && isType(el.type, 'text') && hasCounter(el);
export const hasUnitAndIsTypeTextOrNumber = (el: HTMLInputElement, unit: string): boolean => {
  return el && !!unit && (isType(el.type, 'text') || isType(el.type, 'number'));
};
```